KarhooUISDK/Controllers/Payment/AddCard/AdyenCardRegistrationFlow.swift

```diff
@@ -242,6 +240,10 @@ extension AdyenCardRegistrationFlow: DropInComponentDelegate {
         finish(result: .cancelledByUser)
     }
 
+    func didComplete(from component: DropInComponent) {
```

Comment:
I see we have a `didFail` and a `didCancel` that we are using. This looks like an unhandled success case. Am I missing something? 

Also, cannot comment here directly on the `didCancel` method, but there's a warning there that should be handled.

Reply:
didCancel is fixed.
In terms of didComplete, it is new callback added to delegate. According to documentation: 
https://docs.adyen.com/online-payments/ios/components?tab=v67-actions_1#step-4-additional-client-app
and description in code:
Invoked when the action component finishes, without any further steps needed by the application, for example in case of voucher payment methods. The application just needs to dismiss the DropInComponent.

look like it is called only in case of vouchers (and we are not supporting vouchers, right?). That's why I decided to not put any logic inside this method.